### ISSUE TITLE
trafficserver 9.2.5 FIN sent immediatly on connection to backend server 

### ISSUE THREAD

**Issue by Cyborgscode** [apache/trafficserver#11858]
OS: Fedora 40
Version: 9.2.5

Worked on Fedora 39 with same version, downgrade to fc39 version same release tested, same result.

It does not matter where the backendserver is, local on another port or on a different ip.
This dump is locally on the same server: 

22:48:01.224680 IP 83.246.80.131.53420 > 83.246.80.131.18443: Flags [S], seq 4065112379, win 65495, options [mss 65495,sackOK,TS val 2381864768 ecr 0,nop,wscale 7], length 0
22:48:01.224689 IP 83.246.80.131.18443 > 83.246.80.131.53420: Flags [S.], seq 2155888146, ack 4065112380, win 65483, options [mss 65495,sackOK,TS val 2381864768 ecr 2381864768,nop,wscale 7], length 0
22:48:01.224698 IP 83.246.80.131.53420 > 83.246.80.131.18443: Flags [.], ack 1, win 512, options [nop,nop,TS val 2381864768 ecr 2381864768], length 0
22:48:01.224994 IP 83.246.80.131.53420 > 83.246.80.131.18443: Flags [F.], seq 1, ack 1, win 512, options [nop,nop,TS val 2381864769 ecr 2381864768], length 0
22:48:01.225168 IP 83.246.80.131.18443 > 83.246.80.131.53420: Flags [F.], seq 1, ack 2, win 512, options [nop,nop,TS val 2381864769 ecr 2381864769], length 0
22:48:01.225178 IP 83.246.80.131.53420 > 83.246.80.131.18443: Flags [.], ack 2, win 512, options [nop,nop,TS val 2381864769 ecr 2381864769], length 0

Immediatly after the SYN-ACK is ACKed ATS sends a FIN package. And than it logs:

20241113.22h48m01s CONNECT: attempt fail [CONNECTION_ERROR] to 83.246.80.131:18443 for host='cloud-foo.de' connection_result=Invalid argument [22] error=Invalid argument [22] attempts=3 url='https://cloud-foo.de:18443/equery.js'

A connect with curl to that ip:port works flawless..  The cert is the same as ATS would use. 

the remap is unchanged for years now:

map http://cloud-foo.de/ http://cloud-foo.de:18080/
map https://cloud-foo.de/ https://cloud-foo.de:18443/

something is stopping  ats from sending data at all throu that connection. 

the firewall has been disabled , no change. 

new Systemd version:  "systemd-255.13-1.fc40.x86_64" old was systemd-254 

Already tested:

- ulimits for ats process : 1.000.000 open files
- systemd 254 release
- trafficserver release of fedora 39 
- firewalls :  iptables, nftables, firewalld  - clean, clean, not installed 
- httpd checked
- config checked

**Comment by Cyborgscode** [comment]
Forgot to mention:

this is also logged from time to time

20241113.23h15m17s CONNECT: Invalid argument [22] connecting to 83.246.80.131:18443 for host='cloud-foo.de' url='https://cloud-foo.de:18443/favicon.ico' marking down

**Comment by Cyborgscode** [comment]
```
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: +++++++++ Proxy's Request +++++++++
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: -- State Machine Id: 446
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: GET /favicon.ico HTTP/1.1
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:132.0) Gecko/20100101 Firefox/132.0
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: Accept: image/avif,image/webp,image/png,image/svg+xml,image/*;q=0.8,*/*;q=0.5
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: Accept-Language: de-DE,en-US;q=0.7,en;q=0.3
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: Referer: https://cloud-foo.de/Securityprodukte
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: dnt: 1
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: Cookie: PHPSESSID=j5meul8v4s00b9jof2amo43rkq
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: sec-fetch-dest: image
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: sec-fetch-mode: no-cors
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: sec-fetch-site: same-origin
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: priority: u=6
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: Pragma: no-cache
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: Cache-Control: no-cache
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: Host: cloud-foo.de:18443
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: Accept-Encoding: gzip
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: Client-ip: 87.123.1.6
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: X-Forwarded-For: 87.123.1.6
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: Via: https/2 traffic_server[d2d5ecad-2513-4fff-b579-89fbec2dfb6e] (ApacheTrafficServer/9.2.5)
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: 
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpSM.cc:7646 (call_transact_and_set_next_state)> (http) [446] State Transition: SM_ACTION_API_OS_DNS -> SM_ACTION_CACHE_ISSUE_WRITE
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpSM.cc:4979 (do_cache_prepare_action)> (http_cache_write) [446] writing to cache with URL https://cloud-foo.de:18443/favicon.ico
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpCacheSM.cc:160 (state_cache_open_write)> (http_cache) [446] [&HttpCacheSM::state_cache_open_write, CACHE_EVENT_OPEN_WRITE/TS_EVENT_CACHE_OPEN_WRITE]
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpSM.cc:2741 (main_handler)> (http) [446] CACHE_EVENT_OPEN_WRITE/TS_EVENT_CACHE_OPEN_WRITE, 1108
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpSM.cc:2552 (state_cache_open_write)> (http) [446] [&HttpSM : state_cache_open_write, CACHE_EVENT_OPEN_WRITE/TS_EVENT_CACHE_OPEN_WRITE]
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpTransact.cc:3274 (handle_cache_write_lock)> (http_trans) Next action next; nullptr
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpSM.cc:7646 (call_transact_and_set_next_state)> (http) [446] State Transition: SM_ACTION_CACHE_ISSUE_WRITE -> SM_ACTION_ORIGIN_SERVER_OPEN
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpSM.cc:5096 (do_http_server_open)> (http_track) [446] entered inside do_http_server_open ][ipv4]
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpSM.cc:5129 (do_http_server_open)> (http) [446] open connection to cloud-foo.de: 83.246.80.131:18443
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpSM.cc:5132 (do_http_server_open)> (http_seq) [446] Sending request to server
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpTransact.cc:6654 (will_this_request_self_loop)> (http_transact) [446] max_proxy_cycles = 0
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpTransact.cc:6660 (will_this_request_self_loop)> (http_transact) [446] dst_port = 18443 local_port = 443
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpTransact.cc:6713 (will_this_request_self_loop)> (http_transact) [446] count = 0 <= max_proxy_cycles = 0 : allowing loop
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpSessionManager.cc:403 (_acquire_session)> (http_ss) [acquire session] thread pool search failed
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpSM.cc:5507 (do_http_server_open)> (http) [446] calling sslNetProcessor.connect_re
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpSM.cc:2741 (main_handler)> (http) [446] NET_EVENT_OPEN/TS_EVENT_NET_CONNECT, 200
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpSM.cc:1879 (state_http_server_open)> (http_track) [446] entered inside state_http_server_open: NET_EVENT_OPEN/TS_EVENT_NET_CONNECT
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpSM.cc:1880 (state_http_server_open)> (http) [446] [&HttpSM::state_http_server_open, NET_EVENT_OPEN/TS_EVENT_NET_CONNECT]
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <Http1ServerSession.cc:94 (new_connection)> (http_ss) [465] session born, netvc 0x149338028d80
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <HttpSM.cc:1909 (state_http_server_open)> (http) [446] setting handler for TCP handshake
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.592] [ET_NET 3] DEBUG: <Http2CommonSession.cc:250 (do_start_frame_read)> (http2_cs) [448] receiving frame header
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <Http2CommonSession.cc:266 (do_start_frame_read)> (http2_cs) [448] frame header length=4, type=8, flags=0x0, streamid=15
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <Http2ConnectionState.cc:856 (rcv_window_update_frame)> (http2_con) [448] [15] Received WINDOW_UPDATE frame - updated to: 12582912 delta: 12451840
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpSM.cc:2741 (main_handler)> (http) [446] EVENT_ERROR/TS_EVENT_ERROR/VC_EVENT_ERROR, 3
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpSM.cc:1879 (state_http_server_open)> (http_track) [446] entered inside state_http_server_open: EVENT_ERROR/TS_EVENT_ERROR/VC_EVENT_ERROR
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpSM.cc:1880 (state_http_server_open)> (http) [446] [&HttpSM::state_http_server_open, EVENT_ERROR/TS_EVENT_ERROR/VC_EVENT_ERROR]
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <Http1ServerSession.cc:121 (do_io_close)> (http_ss) [465] session close: nevtc 0x149338028d80
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpTransact.cc:3460 (HandleResponse)> (http_trans) [446] Entering HttpTransact::HandleResponse
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpTransact.cc:3461 (HandleResponse)> (http_seq) [446] Response received
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpTransact.cc:3468 (HandleResponse)> (http_trans) [446] response_received_time: 1731537646
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: +++++++++ Incoming O.S. Response +++++++++
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: -- State Machine Id: 446
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpTransact.cc:3483 (HandleResponse)> (http_seq) [446] Response not valid
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpTransact.cc:3773 (handle_response_from_server)> (http_trans) [446] (hrfs)
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpTransact.cc:3811 (handle_response_from_server)> (http_trans) [446] max_connect_retries: 0 s->current.attempts: 0
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpTransact.cc:3903 (error_log_connection_failure)> (http_trans) [446] [0] failed to connect [5] to 83.246.80.131
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpTransact.cc:3851 (handle_response_from_server)> (http_trans) [446] Error. No more retries.
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpTransact.cc:3974 (handle_server_connection_not_open)> (http_trans) [446] (hscno)
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpTransact.cc:3975 (handle_server_connection_not_open)> (http_seq) [446] Entering HttpTransact::handle_server_connection_not_open
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpSM.cc:5670 (mark_host_failure)> (http) [446] hostdb update marking IP: 83.246.80.131:18443 as down
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpSM.cc:4484 (do_hostdb_update_if_necessary)> (http) [446] server info = 83.246.80.131:18443
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpTransactHeaders.cc:1151 (add_server_header_to_response)> (http_trans) Adding Server: ATS/9.2.5
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: +++++++++ Proxy's Response 2 +++++++++
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: -- State Machine Id: 446
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: HTTP/1.1 502 Invalid argument
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: Date: Wed, 13 Nov 2024 22:40:46 GMT
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: Connection: keep-alive
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: Server: ATS/9.2.5
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: 
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpSM.cc:7646 (call_transact_and_set_next_state)> (http) [446] State Transition: SM_ACTION_ORIGIN_SERVER_OPEN -> SM_ACTION_SEND_ERROR_CACHE_NOOP
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpTunnel.cc:664 (add_producer)> (http_tunnel) [446] adding producer 'internal msg'
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpTunnel.cc:719 (add_consumer)> (http_tunnel) [446] adding consumer 'user agent'
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <HttpTunnel.cc:766 (tunnel_run)> (http_tunnel) tunnel_run started, p_arg is provided
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <Http2ConnectionState.cc:1842 (send_headers_frame)> (http2_con) [448] [15] Send HEADERS frame
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <Http2ConnectionState.cc:1787 (send_a_data_frame)> (http2_con) [448] [15] Send a DATA frame - client window con: 12561642 stream: 12582665 payload:   247
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <Http2ConnectionState.cc:1794 (send_a_data_frame)> (http2_con) [448] [15] END_STREAM
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <Http2Stream.cc:382 (change_state)> (http2_stream) [448] [15] Http2StreamState::HTTP2_STREAM_STATE_HALF_CLOSED_REMOTE -> Http2StreamState::HTTP2_STREAM_STATE_CLOSED
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <Http2ConnectionState.cc:1827 (send_data_frames)> (http2_con) [448] [15] Shutdown stream
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <Http2Stream.cc:505 (initiating_close)> (http2_stream) [448] [15] initiating_close
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.593] [ET_NET 3] DEBUG: <Http2Stream.cc:532 (initiating_close)> (http2_stream) [448] [15] handle write from destroy (event=103)
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.594] [ET_NET 3] DEBUG: <HttpTunnel.cc:1398 (consumer_handler)> (http_tunnel) [446] consumer_handler [user agent VC_EVENT_WRITE_COMPLETE/TS_EVENT_VCONN_WRITE_COMPLETE]
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.594] [ET_NET 3] DEBUG: <HttpSM.cc:3366 (tunnel_handler_ua)> (http) [446] [&HttpSM::tunnel_handler_ua, VC_EVENT_WRITE_COMPLETE/TS_EVENT_VCONN_WRITE_COMPLETE]
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.594] [ET_NET 3] DEBUG: <HttpSM.cc:2741 (main_handler)> (http) [446] HTTP_TUNNEL_EVENT_DONE, 2301
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.594] [ET_NET 3] DEBUG: <HttpSM.cc:3036 (tunnel_handler)> (http) [446] [&HttpSM::tunnel_handler, HTTP_TUNNEL_EVENT_DONE]
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.594] [ET_NET 3] DEBUG: <HttpSM.cc:8605 (clear)> (http_redirect) [PostDataBuffers::clear]
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.594] [ET_NET 3] DEBUG: <HttpSM.cc:1563 (state_api_callout)> (http) [446] calling plugin on hook TS_HTTP_TXN_CLOSE_HOOK at hook 0x563bd492dc60
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.594] [ET_NET 3] DEBUG: <HttpSM.cc:1449 (state_api_callback)> (http) [446] [&HttpSM::state_api_callback, HTTP_API_CONTINUE/TS_EVENT_HTTP_CONTINUE]
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.594] [ET_NET 3] DEBUG: <HttpSM.cc:1489 (state_api_callout)> (http) [446] [&HttpSM::state_api_callout, HTTP_API_CONTINUE/TS_EVENT_HTTP_CONTINUE]
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.594] [ET_NET 3] DEBUG: <HttpSM.cc:7342 (kill_this)> (http_seq) [446] Logging transaction
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.594] [ET_NET 3] DEBUG: <HttpSM.cc:7392 (kill_this)> (http) [446] deallocating sm
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.594] [ET_NET 3] DEBUG: <Http2Stream.cc:70 (~Http2Stream)> (http2_stream) [448] [15] Destroy stream, sent 247 bytes
Nov 13 23:40:46 s113.resellerdesktop.de traffic_manager[12598]: [Nov 13 23:40:46.594] [ET_NET 3] DEBUG: <Http2ConnectionState.cc:1572 (delete_stream)> (http2_con) [448] [15] Delete stream

```

**Comment by jeredfloyd** [comment]
This is #11465 and I've just pushed a backport of the fix in 10.x to the f40 repo.   You can get that build (and provide feedback) here: https://bodhi.fedoraproject.org/updates/FEDORA-2024-b3c4e8da81

**Comment by Cyborgscode** [comment]
omg... it works \o/

**Comment by Cyborgscode** [comment]
I can give the package a +1 karma, Fedoras Infrastructure Login is (again) down :(

**Comment by jeredfloyd** [comment]
Yay!

**Comment by bryancall** [comment]
This was also fixed in the 9.2.7 release.  Redhat let us know about this one and backported the fix to their 9.2.6 package.
